pkg/clustermesh/config.go

```diff
@@ -98,9 +99,19 @@ func (cdw *configDirectoryWatcher) handle(abspath string) {
 		// when the underlying file gets updated, if path points to a symbolic link.
 		// This is required to correctly detect file modifications when the folder
 		// is mounted from a Kubernetes ConfigMap/Secret.
-		if err := cdw.watcher.Add(abspath); err != nil {
+		if err := cdw.watcher.Add(abspath); err != nil && !errors.Is(err, os.ErrNotExist) {
```

Comment:
Nit: We can use `os.IsNotExist` for the same purpose.

Reply:
The godoc of `os.IsNotExist` mentions *"This function predates errors.Is. It only supports errors returned by the os package. New code should use errors.Is(err, fs.ErrNotExist)"*, hence the current approach should be preferred. `os.ErrNotExist` is an alias of `fs.ErrNotExist`.